tools/codegen/gen_backend_stubs.py

```diff
@@ -57,8 +69,8 @@ def native_to_external(
         else:
             assert_never(g)
     for op_name in metadata.keys():
-        if op_name not in native_functions_map:
-            raise AssertionError(f"Found an invalid operator name: {op_name}")
+        assert op_name in native_functions_map, f"Found an invalid operator name: {op_name}. \
+For the list of valid operator names, see https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml"
```

Comment:
Probably wouldn't link it here, because master might not be the same as the version release

Reply:
bleh yeah that's fair